.clabot

```diff
@@ -0,0 +1,521 @@
+{
+  "message": "We require contributors to sign our Contributor License Agreement, and we don't have one on file for {{usersWithoutCLA}}. In order for us to review and merge your code, please e-sign the PDF at https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement.  We then need to manually verify your signature.  We will ping the bot to refresh the PR status when we have confirmed your signature.",
```

Comment:
This isn't used anywhere, can we remove it?

Reply:
I think so, it's a bit weird to support something obviously focused on the old clabot project anyways. Good call.